libs/application-generic/src/services/resource-validator.service.ts

```diff
@@ -12,6 +12,14 @@ import { NotificationStep } from '../usecases';
 import { FeatureFlagsService } from './feature-flags';
 
 export const DAY_IN_MS = 24 * 60 * 60 * 1000;
+const DEMO_WORKFLOWS_IDENTIFIER = [
```

Comment:
We need a more robust mechanism to identify demo workflows. What if as a user I delete the demo workflows and then create new ones with the same `workflowId`?

Reply:
that is a valid point i thought about creating a special tag for those templates, but at the moment it will not fix the problem as we still will need to migrate the old workflows.

at the current version we do not have this problem, even if a workflow was created from the template store it should be counted in the validation.